codalab/apps/web/templates/web/my/index.html

```diff
@@ -92,17 +92,20 @@
     });
 
 
-    $('.competition-publish-button').on('click', function(event) {
+    $('.competition-publish-button').on('click', function (event) {
         event.preventDefault();
         var button = $(this);
-        $.get($(this).prop('href'))
-            .success(function() {
-                button.siblings('.competition-unpublish-button').removeClass('hide');
-                button.addClass('hide');
-            })
-            .error(function() {
-                alert('Error publishing competition, check your Internet connection. If problem persists contact an administrator.');
-            });
+        if (confirm("Are you sure you wish to publish this competition?")) {
```

Comment:
could change this to just `return false` instead of wrapping the rest of the function